### PR TITLE
updated minimum webview2 runtime version and corewebview2initializati…

### PR DESCRIPTION
**Pull request by pagoe-msft**
…on completed

### CODE REVIEW

File: microsoft-edge/webview2/releasenotes.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Release Date: November 23, 2020
 
 * Enabled WinForms designer in .NET Core 3.1+ and .NET 5.  
 * Improved .NET cookie management.  \([\#611][GithubMicrosoftedgeWebviewfeedbackIssue611]\).  
+* Renamed CoreWebView2Ready to [CoreWebView2InitializationCompleted](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2initializationcompletedeventargs?view=WebView2-dotnet-1.0.707-prerelease).
```
Comment (Reezaali) [inline microsoft-edge/webview2/releasenotes.md:50]: @pagoe-msft  can you fix the link please? Thanks.

Comment (pagoe-msft) [reply microsoft-edge/webview2/releasenotes.md:50]: link should be relative path?

Comment (jm-trd-ms) [reply microsoft-edge/webview2/releasenotes.md]: Updated link and verified that the link exists on the live branch.

**Review by jm-trd-ms — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**